packages/react-composites/src/composites/ChatComposite/adapter/AzureCommunicationAttachmentUploadAdapter.ts

```diff
@@ -65,7 +69,9 @@ class AttachmentUploadContext {
 
   public updateAttachmentUpload(
     id: string,
-    data: Partial<Pick<AttachmentMetadata, 'progress' | 'id' | 'name' | 'extension' | 'uploadError' | 'url'>>
+    data: Partial<
+      Pick<AttachmentMetadataWithProgress, 'progress' | 'id' | 'name' | 'extension' | 'uploadError' | 'url'>
```

Comment:
Do the list of pick strings need to be revised?

Reply:
nope, unless we get new feedback to rename any properties